lib/src/__tests__/DatePickerTestingLib.test.tsx

```diff
@@ -78,4 +78,24 @@ describe('<DatePicker />', () => {
 
     expect(screen.queryByRole('dialog')).toBeInTheDocument();
   });
+
+  it('Does not call onChange if same date selected', async () => {
+    const onChangeMock = jest.fn();
+
+    render(
+      <DesktopDatePicker
+        TransitionComponent={FakeTransitionComponent}
+        disableCloseOnSelect
```

Comment:
Is this prop required for the test to fail without the patch?

Reply:
It is not.